sql/pre_install/tables.sql

```diff
@@ -252,6 +252,7 @@ CREATE TABLE IF NOT EXISTS _timescaledb_catalog.continuous_agg (
     direct_view_name NAME NOT NULL,
     max_interval_per_job BIGINT NOT NULL,
     ignore_invalidation_older_than BIGINT NOT NULL DEFAULT BIGINT '9223372036854775807',
+    materialized_only BOOL NOT NULL DEFAULT true,
```

Comment:
won't this cause upgrade failures?

Reply:
the default in the catalog has to be true because all caggs created before the flag/feature got added are materialized only view

Reply:
ok.

Reply:
actually it doesnt have to be true but then the upgrade script needs to set it to true for all existing entries, maybe i should change it to that then the default would match the default behaviour when creating new cagg